overlord/devicestate/devicestate_test.go

```diff
@@ -2224,3 +2224,31 @@ func (s *deviceMgrSuite) TestReloadRegistered(c *C) {
 	}
 	c.Check(ok, Equals, true)
 }
+
+func (s *deviceMgrSuite) TestMarkSeededInConfig(c *C) {
+	st := s.state
+	st.Lock()
+	defer st.Unlock()
+
+	// not seeded -> no config is set
+	s.state.Unlock()
+	s.mgr.Ensure()
```

Comment:
this will start  a first boot change as well?   that might be annoying, not sure

Reply:
Indeed, good point. I updated the test to pretend seeding is already happening and also setting the Serial to avoid device registration.